test/onnx/test_verification.py

```diff
@@ -0,0 +1,67 @@
+# Owner(s): ["module: onnx"]
+
+import unittest
+
+import torch
+from torch.onnx import verification
+
+
+class TestVerification(unittest.TestCase):
+    def test_check_jit_model_diff(self):
```

Comment:
I would include the expected behavior as part of the test method name. To make what a test is testing clear: `test_<method or function name>_<expected behavior>_[when_<condition>]`.

e.g. `test_method_x_raises_error_when_dims_is_not_a_sequence`

Reply:
Same for other test methods